components/NavigationDrawer/src/MDCBottomDrawerPresentationController.m

```diff
@@ -76,7 +72,8 @@ - (void)presentationTransitionWillBegin {
   self.bottomDrawerContainerViewController.delegate = self;
 
   self.scrimView = [[UIView alloc] initWithFrame:self.containerView.bounds];
-  self.scrimView.backgroundColor = DrawerOverlayBackgroundColor();
+  self.scrimView.backgroundColor =
+      self.scrimColor ?: [UIColor colorWithWhite:0 alpha:(CGFloat)0.32];
```

Comment:
was the scrim alpha change on purpose?

Reply:
Yes

Reply:
this is a breaking visual change, please mark it as one for the next release cop to know.

Reply:
Done